pkg/webui/components/input/generate.js

```diff
@@ -41,13 +41,12 @@ const GenerateInput = props => {
     () => ({
       icon: 'autorenew',
       type: 'button',
-      title: generateTitle,
```

Comment:
By deleting this title, cypress cannot find the button by `{ name: 'Generate'}`. So that's why the front-end test is failing. Can you please update the test